### PR TITLE
adding test.html

### PR DESCRIPTION
**Pull request by BenJMatthies**

### CODE REVIEW

**Comment by Lisantim** [comment]
Looks good to me.

**Comment by evdotjs** [comment]
great!

File: test.html
Comment (awakenjourney) [inline test.html:1]: great!

**Review by awakenjourney — COMMENTED** [review]
great!

File: test.html
Comment (Lisantim) [reply test.html:1]: Good!

File: test.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>This is a test</title>
+</head>
+<body>
+    
+</body>
+</html>
```
Comment (evdotjs) [inline test.html:12]: 👍️

**Review by Lisantim — APPROVED** [review]

**Review by evdotjs — APPROVED** [review]

**Pull request merged**